apps/src/pythonlab/pyodideWebWorker.js

```diff
@@ -20,6 +20,7 @@ async function loadPyodideAndPackages() {
       'numpy',
       'matplotlib',
       `/blockly/js/pyodide/${version}/pythonlab_setup-0.0.1-py3-none-any.whl`,
+      `/blockly/js/pyodide/${version}/unittest_runner-0.0.1-py3-none-any.whl`,
```

Comment:
I wouldn't mind linking to the unittest_runner repo here since its outside the mono-repo, I can imagine somebody struggling to figure out where this code came from in several years.

Reply:
Also... is there a reason to keep our python packages outside the mono-repo? I could see it hard finding a home for them, but there's no particular reason we have to keep "non ruby/js" code elsewhere. Having them in-repo would allow PRs like this one to be a single PR, so you can make lock-step changes to both sides of the "python wall" at the same time, and would improve searching. I don't think its a factor for this PR, but might be worth discussing out of band?

Reply:
> I could see it hard finding a home for them

This was my main reason, afaik we don't have any python code in our repo, and this code would exist in some space other than apps or dashboard. I am happy to put them in the main repo if we have a good idea as to where that would be, but it seemed cleaner for now to have them split out. Agree it's worth discussing!

Reply:
Also added link back to repo, good idea!